### PR TITLE
Create announce commands and method

### PR DESCRIPTION
**Pull request by DUFCLiam**
Create announce commands and method
***
**Associated issue:** 
"/announce" (#803) 

**Why should we merge this:**
We should merge this because it has a good potential on the server because we can pretend to be announcements because if we want to announce something we can do it without having to make a PR or use /me and mimic the announcement prefix and the purple asterisk puts people off reading because they know it's fake and do not respond/notice it.

**Do you think it will be used?:**
Yes, as us admins want to announce things to the server with the announcement prefix it can have more use.

**Does this make a current feature more used?:**
Yes, It makes the announcement prefix more used as it is not widely used on this plugin.
***
Thank _you_ for reading this I hope you have a good day/night.

### CODE REVIEW

**Comment by LegendIsAwesomes** [comment]
This basically  mimics /say.

**Comment by DUFCLiam** [comment]
As you said in the other one, Discuss that in the associate issue. Not the Pull Request @LegendIsAwesomes

**Comment by LegendIsAwesomes** [comment]
I'm talking about how your code mimics /say.

**Comment by DUFCLiam** [comment]
Well, It announces a message that's all the associated issue covers if you can make a suggestion I will happily add it @LegendIsAwesomes

**Comment by LegendIsAwesomes** [comment]
The formatting is a slight bit off, also what's with the random comment.

**Comment by DUFCLiam** [comment]
I will fix the formatting.

I used the comment because if you are scanning more than one method you search "//" in your IDE and it will search it. (I accidentally added it)

**Comment by LegendIsAwesomes** [comment]
The comment isn't needed here though, I knew about searching for methods.

**Comment by DUFCLiam** [comment]
Please comment on any remaining formatting faults now.

**Comment by LegendIsAwesomes** [comment]
Should be good to go now.

**Comment by DUFCLiam** [comment]
Thank you for your help towards this.

**Comment by LegendIsAwesomes** [comment]
Oh and why is there a method called announce, a method isn't needed.

**Comment by DUFCLiam** [comment]
Actually @LegendIsAwesomes Jerom asked for this to be added:
>

Don't we have TFM_Announcer.getPrefix()?
Since we're at it, why not make a method: TFM_Announcer.announce(String)?

**Comment by LegendIsAwesomes** [comment]
Yes, but this makes command_announce.announce (string); not TFM_Announcer.

**Comment by DUFCLiam** [comment]
Oh yeah xD I have to go to utils :P

**Comment by DUFCLiam** [comment]
Anymore input?

File: src/me/StevenLawson/TotalFreedomMod/TFM_Util.java [hunk 1]
```diff
@@ -129,6 +129,16 @@ public static void bcastMsg(String message)
     {
         TFM_Util.bcastMsg(message, null);
     }
+    
+    public static void announceMsg(String message, ChatColor color)
```
Comment (AvalancheYT) [inline src/me/StevenLawson/TotalFreedomMod/TFM_Util.java]: I don't know why but why did you add this?

File: src/me/StevenLawson/TotalFreedomMod/TFM_Util.java [hunk 2]
```diff
@@ -129,6 +129,16 @@ public static void bcastMsg(String message)
...
+    {
+        TFM_Util.bcastMsg(TFM_ConfigEntry.ANNOUNCER_PREFIX.getString().replaceAll("&", "§") + message);
+    }
+    
+    public static void announceMsg(String message)
```
Comment (AvalancheYT) [inline src/me/StevenLawson/TotalFreedomMod/TFM_Util.java]: Same as above

File: src/me/StevenLawson/TotalFreedomMod/TFM_Util.java (hunk 1 above)
Comment (DUFCLiam) [inline src/me/StevenLawson/TotalFreedomMod/TFM_Util.java]: So you can do "announceMsg("Hello World!");" and it will be like an announcement.
